dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/JarLinkageReport.java

```diff
@@ -131,6 +131,13 @@ public String formatByGroup() {
               .collect(Collectors.joining(", "));
 
       builder.append("Referenced by: ");
+
+      // Group is marked unreachable if none of the errors in the group is reachable
```

Comment:
What is a group in this context?

Reply:
It's LinkageErrorCause.

Added it in comment.